# 8x3 matrix stored row-major
float64[24] data